sdb/commands/container_of.py

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright 2020 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pylint: disable=missing-docstring
+
+#
+# The pylint workaround below is for the example
+# sections of the help messages.
+#
+# pylint: disable=line-too-long
+
+import argparse
+from typing import Iterable
+
+import drgn
+import sdb
+from sdb.commands.internal.util import get_valid_struct_name
+
+
+class ContainerOf(sdb.Command):
+    """
+    Get the containing object of a pointer object.
+
+    EXAMPLES
+        Superfluous example of getting a structure's address
```

Comment:
`Superfluous` -> `Trivial` ?  If it's really superfluous then we can remove it :)